testing/integration/otel_helm_test.go

```diff
@@ -128,108 +147,6 @@ func TestOtelKubeStackHelm(t *testing.T) {
 	}
 }
 
-func TestOtelKubeStackHelmEDOTImage(t *testing.T) {
-	info := define.Require(t, define.Requirements{
-		Stack: &define.Stack{},
-		Local: false,
-		Sudo:  false,
-		OS: []define.OS{
-			// only test the basic and the wolfi container with otel
-			{Type: define.Kubernetes, DockerVariant: "elastic-otel-collector"},
-			{Type: define.Kubernetes, DockerVariant: "elastic-otel-collector-wolfi"},
```

Comment:
Moved these two cases to `TestOtelKubeStackHelm` to reduce duplicated code. Let me know if they still need to run in a separate test.